### ISSUE TITLE
Fix bakkes.sh if

### ISSUE THREAD

**Issue by CrumblyLiquid** [CrumblyLiquid/BakkesLinux#7]

**Comment by Ciflire** [comment]
Hi,
the `install.sh` is broken [here](https://github.com/CrumblyLiquid/BakkesLinux/blob/master/install.sh#L34) you might wanna change that line into `WINEDEBUG=-all protontricks -c "wine $E_ZIP/BakkesModSetup.exe" 252950`, specifying that it should be launched with wine should fix the issue with it.
Also, you might want to change [this line](https://github.com/CrumblyLiquid/BakkesLinux/blob/master/install.sh#L37) to use regex for steam/steamwebhelper

**Comment by CrumblyLiquid** [comment]
Hi!
`install.sh` should now actually launch the executable. Thanks for the fix!
I also added some improvements to `install.sh` and `bakkes.sh`.

As for changing `while killall -0 steam &> /dev/null; do` I'm not sure why.

**Comment by Ciflire** [comment]
Nice work, your installation script works out of the box, i just tried it on a fresh install. Everything works fine, just had to set the launch options, it seems that the `localconfig.vdf`is located under `.local/share/Steam/userdata/[userId different from 0]/config/localconfig.vdf` . On a friends computer it's also located in that path, with a different userId.

For the second point, if I understood correctly the point of this line, it's to kill steam, to be sure it's not trying to use the vdf file, but killing the steam executable is not enough, most of the time, killing `steamwebhelper` does actually kill the thing. Right now, it is stuck on setting the launch options because of that, until i manually kill Steam. But I might be mistaken on what you wanted to do here.

**Comment by CrumblyLiquid** [comment]
Thanks!

> Everything works fine, just had to set the launch options, it seems that the `localconfig.vdf`is located under `.local/share/Steam/userdata/[userId different from 0]/config/localconfig.vdf`. On a friends computer it's also located in that path, with a different userId.

The problem is that `modcfg.py` fails to find the correct path?

> For the second point, if I understood correctly the point of this line, it's to kill steam, to be sure it's not trying to use the vdf file

It's just a waiting for Steam to exit (not actually killing Steam).

I don't really want to kill Steam unpromted so I just send it signal 0 (doesn't send anything but still does error checking) to test if it's running or not. If it's not running `killall` returns non-zero value and the while loop stops running.

**Comment by Ciflire** [comment]
> The problem is that `modcfg.py` fails to find the correct path?

All right I miss one info. It does not work for me because I have to user accounts for some reasons. But I saw that you left a `TODO` in your code for multiple users. For singleuser users I just assume it'll work properly. Great work again and thank you for your work!

> It's just a waiting for Steam to exit (not actually killing Steam).
> 
> I don't really want to kill Steam unpromted so I just send it signal 0 (doesn't send anything but still does error checking) to test if it's running or not. If it's not running `killall` returns non-zero value and the while loop stops running.

Ok np. Maybe a more explicit message could be user friendly smth like "Please close steam for the installation process to resume" ? That's my impression, maybe some external input could be good too

**Comment by CrumblyLiquid** [comment]
> It does not work for me because I have two user accounts for some reasons.

Yeah I'll have to borrow another account to test this. I'm not sure when I get around to that.
In the meantime one could modify [this line](https://github.com/CrumblyLiquid/BakkesLinux/blob/3e21096d96a92f97a5de3517280e7a4f8b434b3f/modcfg.py#L16) to point to the correct folder.
Something like `userid = Path(steam_path, "userdata/<your_id>")`

> Maybe a more explicit message could be more user friendly smth like "Please close steam for the installation process to resume" ?

Fixed here 3e21096d96a92f97a5de3517280e7a4f8b434b3f
